src/layouts/components/Editor/extensions/Cards/Cards.ts

```diff
@@ -154,9 +154,13 @@ export const IsomerCards = Node.create<CardOptions>({
         if (dispatch) {
           const offset = tr.selection.anchor + 1
 
-          tr.replaceSelectionWith(node)
+          tr.insert(tr.selection.anchor, node)
```

Comment:
May I get context behind this change?

Reply:
The thinking is that when users click on add card grid, they don't expect their current contents to be removed. The `replaceSelectionWith` function does a removal of the existing selection, which would be potentially unexpected for the user.

Simply put, adding content is not the same as replacing content, so this change is to respect that mental model.